p11-kit/test-server.c

```diff
@@ -101,19 +108,23 @@ setup_server (void *arg)
 	if (envvar)
 		setenv ("LD_PRELOAD", envvar, 1);
 
-	argv[5] = test.socket_path;
-	argv[6] = (char *)arg;
+	p11_array_push (args, "-n");
+	p11_array_push (args, test.socket_path);
+	p11_array_push (args, fixture->token);
+	p11_array_push (args, NULL);
```

Comment:
error checking missing for `p11_array_push` in the test server

Reply:
Fixed.